dapps/marketplace/src/pages/nav/Profile.js

```diff
@@ -42,10 +43,16 @@ const ProfileNav = ({
           console.error(error)
           return null
         }
-        if (!get(data, 'web3.primaryAccount')) {
+        const accountID = get(data, 'web3.primaryAccount.id')
+        if (!accountID) {
           return null
         }
 
+        // set the user for sentry
+        Sentry.configureScope(scope => {
```

Comment:
I'd suggest to move this out of this HOC. We could add a component (added to root inside App) that uses this HOC and set the scope inside `useEffect` hook of that component.